modules/update-evaluate-availability.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-evaluate-availability_{context}"]
+= Evaluating update availability
+
+The Cluster Version Operator (CVO) periodically queries the OpenShift Update Service (OSUS) for the most recent data about update possibilities.
+This data is based on the channel to which the cluster is subscribed.
+The CVO then saves information about update recommendations and risks into either the `availableUpdates` or `conditionalUpdates` field of its ClusterVersion resource.
+
+The CVO periodically checks the conditional updates for update risks.
+These risks are conveyed through the data served by the OpenShift Update Service, which contains information for each version about known issues that may affect a cluster updated to that version.
+Most risks are limited to clusters with specific characteristics, such as clusters with a certain size or clusters that are deployed in a particular cloud platform.
+
+The CVO continuously evaluates its cluster characteristics against the conditional risk information for each conditional update, and if it finds that the cluster matches the criteria, it stores this information in the `conditionalUpdates` field of its ClusterVersion resource.
+If the CVO finds that the cluster doesn't match the risks of an update, or that there are no risks associated with the update, it stores the target version in the `availableUpdates` field of its ClusterVersion resource.
+
+The user interface, either the web console or the OpenShift CLI (`oc`), then presents this information to the administrator.
+Each "supported but not recommended" update recommendation contains a link to further resources about the risk so that the administrator can make an informed decision about the update.
+
+You can inspect all available updates with the following command:
+
+[source,terminal]
+----
+$ oc adm upgrade --include-not-recommended
+----
+
+The additional `--include-not-recommended` parameter includes updates that are available but not recommended due to a known risk that applies to the cluster.
```

Comment:
I'm not sure I understand this, sorry. 

TLDR: "known risk" refers to an issue for a specific update, not a general issue for all OCP clusters that prevents valid updates from showing up as available.

At any point, there may be a number of OCP updates that are still supported by Red Hat, but are not explicitly recommended because they can have a certain issue that _might_ cause problems for clusters with a specific characteristic. 

For instance, the one "supported but not recommended" update in the example below has the following known risk: "Nodes with substantial numbers of containers and CPU contention may not reconcile machine configuration". Because of this risk, users can only see this available update when they use the `--include-not-recommended` parameter and thus signal that they are willing to consider potentially problematic updates.